### PR TITLE
first functional menu

### PR DESCRIPTION
**Pull request by hawciu**
general working menu flow

### CODE REVIEW

File: Big-Bumper-Bash/Assets/UI/MapChoiceScript.cs [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+
+    }
+
+    public void PressMap1()
```
Comment (Gathaspar66) [inline Big-Bumper-Bash/Assets/UI/MapChoiceScript.cs:22]: In my opinion these 2 methods do exactly the same thing, shouldn't there be one method but with a parameter, what do you think?

**Review by Gathaspar66 — APPROVED** [review]
lgtm!

File: Big-Bumper-Bash/Assets/UI/CarChoiceScript.cs [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    }
+
+    private void OnEnable()
+    {
+        defaultSelected.GetComponent<Button>().Select();
+    }
+
+    public void PressCar(string car = "no car chosen")
+    {
+        print("starting game - car choice: " + car);
```
Comment (masslany) [inline Big-Bumper-Bash/Assets/UI/CarChoiceScript.cs:30]: Unnecessary log

File: Big-Bumper-Bash/Assets/UI/MapChoiceScript.cs [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    }
+
+    private void OnEnable()
+    {
+        defaultSelected.GetComponent<Button>().Select();
+    }
+
+    public void PressMap(string map = "no map chosen")
+    {
+        print("map choice: " + map);
```
Comment (masslany) [inline Big-Bumper-Bash/Assets/UI/MapChoiceScript.cs:30]: Unnecessary log

File: Big-Bumper-Bash/Assets/UI/MapChoiceScript.cs [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    {
+
+    }
+
+    private void OnEnable()
+    {
+        defaultSelected.GetComponent<Button>().Select();
+    }
+
+    public void PressMap(string map = "no map chosen")
```
Comment (masslany) [inline Big-Bumper-Bash/Assets/UI/MapChoiceScript.cs:28]: If there is a determined number of maps it'd be better to use enum to represent them.

File: Big-Bumper-Bash/Assets/UI/CarChoiceScript.cs [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+        //gameObject.SetActive(false);
```
Comment (masslany) [inline Big-Bumper-Bash/Assets/UI/CarChoiceScript.cs:31]: Don't leave commented out code

File: Big-Bumper-Bash/Assets/UI/CarChoiceScript.cs [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+    {
+
+    }
+
+    private void OnEnable()
+    {
+        defaultSelected.GetComponent<Button>().Select();
+    }
+
+    public void PressCar(string car = "no car chosen")
```
Comment (masslany) [inline Big-Bumper-Bash/Assets/UI/CarChoiceScript.cs:28]: This method certainly doesn't press a car. Correct me if I'm mistaken but they're called when a button is pressed. If that's the case then a better naming convention for them is `OnXyzPressed`. This applies to all the methods across the different screen. Also if the cars are in a determined set, it's better to represent them using an enum.
```suggestion
    public void OnChooseCarButtonPressed(string car = "no car chosen")
```

**Review by masslany — CHANGES_REQUESTED** [review]

**Review by masslany — APPROVED** [review]

**Review by klaudix30 — APPROVED** [review]

**Review by Gathaspar66 — APPROVED** [review]
lgtm!

**Pull request merged**